app/validators/employment_details/answers_validator.rb

```diff
@@ -0,0 +1,43 @@
+module EmploymentDetails
```

Comment:
Is this only separate because we thought it might need to be to handle Appeal no changes? That was the only scenario that Employment details was asked separately to Income as far as I knew (and now it's not going to be)
https://docs.google.com/spreadsheets/d/1V5GFO7nlamQZtUyrQX2Rd2PKiL3a_KRK2fSC_AIGOo0/edit#gid=0